html/answers/gaurav-q4.html

```diff
@@ -0,0 +1,72 @@
+<html>
+    <head>
+
+        <!-- Global site tag (gtag.js) - Google Analytics -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157837769-1"></script>
+        <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            HTML | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What is the difference between span and div?
+">
+    </head>
+    <body>
+		
+		<img src="../../assets/CN_logo.png" height="72px" alt="Coding Ninjas Logo" />
+
+        <div>
+            <p>
+                <small>
+
+                    <a href="../../index.html">Home</a>
+                    &gt; <a href="../index.html">HTML Questions</a>
+                    &gt;What is the difference between span and div?
+
+
+                </small>
+               
+            </p>
+        </div>
+
+
+        <content>
+            <div>
+                <h4>
+                    <h4>What is the difference between span and div?
+</h4>
+                </h4>
+            </div>
+            <br>
+            <br>
+            <article>
+                <strong>Answer:</strong> 
+                <p>
+                    The difference is that <code>span</code> gives the output with <code> display: inline </code>and div gives the output with<code> display: block.</code>
```

Comment:
Add an example for both span and div.

Reply:
ok. on it.

Reply:
Made changes @shivamyth . pls review again.